### PR TITLE
Performers expand on click. Sloppy and needs formatting. 

### PR DESCRIPTION
**Pull request by clendon**
I am going to reformate and restructure the grid, divs, etc. Just wanted to make sure the _functional_ version of the repo was at least up and running.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -26,5 +26,7 @@ module.exports = {
     ],
     "rules": {
         "no-console": "off",
+        "no-underscore-dangle": "off",
+        "react/prop-types": "off",
```
Comment (clendon) [inline .eslintrc.js:30]: Disabling more rules to make life easier for us all.

**Comment by haydenbp** [comment]
image
Nothing happens when i search for someone. list is blank

**Review by Joshua-Barnes03 — APPROVED** [review]
I have this currently pulled down, up and active. All working.

**Review by haydenbp — APPROVED** [review]
Looks good, everything seems to be working

**Pull request merged**